disasterinfosite/migrations/0001_initial.py

```diff
@@ -129,23 +129,23 @@ class Migration(migrations.Migration):
                 ('order_of_appearance', models.IntegerField(default=0, help_text="The order in which you'd like this to appear in the tab. 0 is at the top."))
             ],
         ),
-        migrations.CreateModel(
-            name='SnuggetSubSection',
+         migrations.CreateModel(
+            name='SnuggetPopOut',
             fields=[
-                ('id', models.AutoField(primary_key=True, verbose_name='ID', auto_created=True, serialize=False)),
-                ('name', models.CharField(max_length=50)),
-                ('display_name', models.CharField(default="", help_text='The name to show for this section', max_length=50)),
-                ('order_of_appearance', models.IntegerField(default=0, help_text="The order in which you'd like this to appear in the section. 0 is at the top. These can be in different sections or mutually exclusive, hence the non-unique values."))
+                ('id', models.AutoField(auto_created=True, primary_key=True, serialize=False, verbose_name='ID')),
+                ('alt_text',models.TextField(default='', max_length=255)),
+                ('image', models.ImageField(upload_to='popout_images')),
+                ('link', models.TextField(default='', max_length=255)),
+                ('text', models.TextField(default='')),
```

Comment:
Is there a functional difference between `models.CharField` and `models.TextField`, or is this just terminology changing with successive Django versions?

Reply:
Never mind, I just found an explanation at https://stackoverflow.com/questions/7354588/django-charfield-vs-textfield
Based on that, this is a change for the better.  I've never noticed any advantage that postgres `varchar` offers over `text`, and it gives us more flexibility for longer content.
Mostly, I wanted to make sure this wasn't going to cause non-Latin-script problems, and now I'm satisfied that it won't.